docs/source/pages/arducam.rst

```diff
@@ -9,16 +9,29 @@ Overview
 ********
 
 `ArduCam <https://www.arducam.com/product-category/opencv-ai-kit/>`__ has built a series of M12-capable and C/CS-capable camera boards that work with DepthAI
-and megaAI. A variety of M12-mount options allow to match lenses in many projects. More about how to get different field of view `here <https://docs.luxonis.com/en/latest/pages/faq/#how-do-i-get-different-field-of-view-or-lenses-for-depthai-and-megaai>`__.
```

Comment:
Shall we update product names here?

Reply:
Yes please do.  We need to scrub the FAQ/etc. as well.